sys-devel/gdb/gdb-11.1.ebuild

```diff
@@ -193,6 +193,9 @@ src_configure() {
 	# source-highlight is detected with pkg-config: bug #716558
 	export ac_cv_path_pkg_config_prog_path="$(tc-getPKG_CONFIG)"
 
+	# ensure proper compiler is detected for Clang bulids: bug #831202
```

Comment:
```suggestion
	# ensure proper compiler is detected for Clang builds: bug #831202
```

Reply:
@thesamesam Thanks for catching this! I force pushed the fix so GH won't add an additional commit to keep the log clean.